libkineto/src/CudaDeviceProperties.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "CudaDeviceProperties.h"
+
+#include <vector>
+
+#include <cuda_runtime.h>
+#include <cuda_occupancy.h>
+
+namespace KINETO_NAMESPACE {
+
+std::vector<cudaOccDeviceProp> occProps;
```

Comment:
static or anon namespace?

Reply:
The previous "occProps" is a static variable which does not expose to other files. Now removed and put to static variable inside function "occDeviceProps".